source/type/data.cpp

```diff
@@ -61,7 +61,7 @@ namespace pt = boost::posix_time;
 
 namespace navitia { namespace type {
 
-const unsigned int Data::data_version = 69; //< *INCREMENT* every time serialized data are modified
+const unsigned int Data::data_version = 70; //< *INCREMENT* every time serialized data are modified
```

Comment:
In fact you do not need to change `data_version` as the `data.nav.lz4` doesn't contain any `Impact`.
The serializer of `Impact` is only used during a clone, inside a process, but never to be persisted.

Reply:
OK, I'm fine with both.
But it's also saving a lot of time at deploy saving re-bina on every coverage to stay on the same version... So I'll go back to 69 on that.

Reply:
Nice catch @kinnou02. Go back to `Data::data_version = 69`